polling/src/main/java/io/atleon/polling/reactive/PolledRecord.java

```diff
@@ -0,0 +1,34 @@
+package io.atleon.polling.reactive;
+
+import java.util.Optional;
+
+/**
+ * The type of the records that are produced by the {@link Pollable}.
+ * @param <P> The type of the payload of the events.
+ * @param <O> The type of the object used for determining offsets.
+ */
+public interface PolledRecord<P, O> {
```

Comment:
I wonder if the suffix `Record` here is a meaningful qualification. Isn't this really just "something that was `Polled`?". Maybe `Record` implies that there's a payload and an offset?

Reply:
That's a good point, naming is hard. I think you're right though, thinking of it from the end user standpoint the `PolledRecord` is really just what the `Pollable` returns so it makes sense to just be `Polled`, I'll make that change.